server/models/Profile.js

```diff
@@ -0,0 +1,54 @@
+const mongoose = require('mongoose');
+
+// values for avaialbility hours each week
+const availability_values = ["10" , "10+" , "20" , "20+" , "30" , "30+"]
```

Comment:
Follow consistent naming convention. For JS, we do recommend follow camelCase so for this case

```suggestion
const availabilityValues = ["10" , "10+" , "20" , "20+" , "30" , "30+"]
```

Reply:
changed to camelCasing